src/main/java/mate/service/DriverServiceImpl.java

```diff
@@ -35,4 +36,9 @@ public Driver update(Driver driver) {
     public boolean delete(Long id) {
         return driverDao.delete(id);
     }
+
+    @Override
+    public Optional<Driver> findByLogin(String login) {
+        return driverDao.findByLogin(login);
```

Comment:
service should return Driver or throw exception that driver wasn't found in db. Optional is returned from dao